resourcesManager/resourcesManager-war/src/main/java/org/silverpeas/components/resourcesmanager/web/ReservationEntity.java

```diff
@@ -92,8 +94,11 @@ public static ReservationEntity createFrom(final String instanceId,
    * Default hidden constructor.
    */
   private ReservationEntity(final String instanceId, final Reservation reservation) {
-    super("reservation", instanceId, reservation.getIdAsString(), reservation.getEvent(), null,
-        new DateTime(reservation.getBeginDate()), new DateTime(reservation.getEndDate()), null);
+    super(ContributionIdentifier.from("reservation", instanceId, reservation.getIdAsString()),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "AbstractEventEntity"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)